src/index.js

```diff
@@ -63,7 +63,7 @@ server.register(
 
   debug('extensions registered')
 
-  if (runtime.login) {
+  if (runtime.login && !process.env.GITHUB_DISABLE_AUTHENTICATION) {
```

Comment:
wouldn't it be better to simply not define `login` in the relevant config file? then no change is needed here...